### PR TITLE
Bug 1248476 - Updates to Akismet seed command et al

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by jezdez** [comment]
@jwhitlock @willkg r?

**Comment by willkg** [comment]
Travis got unhappy with this:

```
kuma/wiki/management/commands/submit_deleted_documents.py:4:1: F401 'models' imported but unused
```

**Comment by jezdez** [comment]
@willkg D'oh.

File: kuma/wiki/management/commands/submit_deleted_documents.py [hunk 1]
```diff
@@ -44,13 +43,11 @@ def handle(self, *args, **options):
         # first get the deleted document logs for the last n days
         ttl = timezone.now() - timedelta(days=options['days'])
         logged_deletions = DocumentDeletionLog.objects.filter(
-            timestamp__gte=ttl,
-        ).filter(
-            # They use "spam" or "junk"
+            # They use "spam"
             # deleting spam revisions;
             # the spam makes me cry.  -- willkg
```
Comment (willkg) [inline kuma/wiki/management/commands/submit_deleted_documents.py]: This totally breaks the haiku, but so it goes.

**Comment by willkg** [comment]
I think this looks fine as far as I can tell. r+ if travis is cool with it.

**Comment by jezdez** [comment]
@willkg thanks!

File: kuma/wiki/management/commands/submit_deleted_documents.py (hunk 1 above)
Comment (jezdez) [inline kuma/wiki/management/commands/submit_deleted_documents.py]: This is the beauty of old code, isn't it?

**Pull request merged**